app.py

```diff
@@ -241,23 +241,14 @@ def init_openai_client(use_data=SHOULD_USE_DATA):
             'x-ms-useragent': USER_AGENT
         }
 
-        if use_data:
-            base_url = f"{str(endpoint).rstrip('/')}/openai/deployments/{deployment}/extensions"
-            azure_openai_client = AsyncAzureOpenAI(
-                base_url=str(base_url),
-                api_version=AZURE_OPENAI_PREVIEW_API_VERSION,
-                api_key=aoai_api_key,
-                azure_ad_token_provider=ad_token_provider,
-                default_headers=default_headers,
-            )
-        else:
-            azure_openai_client = AsyncAzureOpenAI(
-                api_version=AZURE_OPENAI_PREVIEW_API_VERSION,
-                api_key=aoai_api_key,
-                azure_ad_token_provider=ad_token_provider,
-                default_headers=default_headers,
-                azure_endpoint=endpoint
-            )
+        azure_openai_client = AsyncAzureOpenAI(
+            api_version=AZURE_OPENAI_PREVIEW_API_VERSION,
```

Comment:
Can we add a check to ensure that this version is >= 2024-02-15-preview? since this will not be compatible with previous versions anymore

Reply:
Yep, I just added this at the top of the function.  I also included a variable near the top of the file to make it easy for us to remember where to update the value of the min version in the future.  Let me know if you have any other suggestions.